client/main.js

```diff
@@ -45,9 +46,9 @@ Meteor.subscribe('bannedProfiles');
 
 Tracker.autorun(() => {
   if (Meteor.loggingIn() || Meteor.user()) {
-    if (Profiles.findOne({userId: Meteor.userId()}) !== undefined) {
-      if (Profiles.findOne({userId: Meteor.userId()}).banned) {
-        throwError("Login denied. This account is currently disabled.");
+    if (Profiles.findOne({ userId: Meteor.userId() }) !== undefined) {
+      if (Profiles.findOne({ userId: Meteor.userId() }).banned) {
```

Comment:
Why the nested if statements here? It's a bad pattern. Preferably we'd join these with `&&`.

Reply:
Maybe there's a reason to do it this way but IDK